src/parquet/parquet.h

```diff
@@ -40,6 +40,26 @@
 #include <thrift/protocol/TBinaryProtocol.h>
 #include <thrift/transport/TBufferTransports.h>
 
+//macros to disable padding. Required for Int96
```

Comment:
Hence creating a separate JIRA about it :) We will need several unit tests before we go that route as well.